exercises/svm/ic_sol_svm_2.Rnw

```diff
@@ -0,0 +1,20 @@
+% !Rnw weave = knitr
+
+<<setup-child, include = FALSE>>=
+library('knitr')
+knitr::set_parent("../../style/preamble_ueb.Rnw")
+@
+
+\input{../../latex-math/basic-math.tex}
+\input{../../latex-math/basic-ml.tex}
+\input{../../latex-math/ml-svm.tex}
+\input{../../latex-math/ml-gp.tex}
+
+\kopficsl{11}{Support Vector Machines}
```

Comment:
And actually, here is the "Nonlinear" Missing :D

Reply:
not my brightest PR...

Reply:
:)